tests/test_app/TestApp/Application.php

```diff
@@ -26,6 +26,18 @@ class Application extends BaseApplication
      */
     public function bootstrap()
     {
+        $this->addPlugin('TestPlugin',
```

Comment:
What's all this?

Reply:
I have no clue where this comes from...

Reply:
Ah this happens when I run all the test locally and they fail (since I have no DB set up)